plugins/inventory/aws_ec2.py

```diff
@@ -649,7 +700,8 @@ def parse(self, inventory, loader, path, cache=True):
 
         # get user specifications
         regions = self.get_option('regions')
-        filters = ansible_dict_to_boto3_filter_list(self.get_option('filters'))
+        include_filters = [self.get_option('filters')] + self.get_option('include_filters')
+        exclude_filters = self.get_option('exclude_filters')
```

Comment:
Should we deprecate the 'filters' option ?  (can we deprecate inventory parameters?)

Reply:
I don't feel the need for that. The long term support should be rather easy anyway.